### ISSUE TITLE
Add Documentation for amp-to-war profile

### ISSUE THREAD

**Issue by bhagyas** [Alfresco/alfresco-sdk#309]

**Comment by mindthegab** [comment]
Is the command reference for each archetype, now officially maintained in docs, enough?

http://docs.alfresco.com/5.0/concepts/alfresco-sdk-cmd-reference-repo-amp.html

**Comment by bhagyas** [comment]
Looks good.

Can we include some of the basic commands on the README.md on GitHub?

**Comment by mindthegab** [comment]
Honestly I wouldn't want to duplicate information we kind of tried with @gravitonian to separate:

End user docs - In docs.alfresco.com, for Community and officially supported for Enterprise
Github - SDK Developer docs (e.g. developer wiki, see https://github.com/Alfresco/alfresco-sdk/wiki/Developer-Wiki).

Neverthless in commit 016879ec872235966c3214fd79c8719a6bec93c4 I have added a link from the README to the command reference, so hopefully this gives people a pointer, without us having to keep duplicated info up to date. 

Closing this one.
